django/contrib/admin/static/admin/js/autocomplete.js

```diff
@@ -1,28 +1,23 @@
 'use strict';
 {
     const $ = django.jQuery;
-    const init = function($element, options) {
-        const settings = $.extend({
-            ajax: {
-                data: function(params) {
-                    return {
-                        term: params.term,
-                        page: params.page,
-                        app_label: $element.data('app-label'),
-                        model_name: $element.data('model-name'),
-                        field_name: $element.data('field-name')
-                    };
-                }
-            }
-        }, options);
-        $element.select2(settings);
-    };
 
-    $.fn.djangoAdminSelect2 = function(options) {
-        const settings = $.extend({}, options);
+    $.fn.djangoAdminSelect2 = function() {
         $.each(this, function(i, element) {
             const $element = $(element);
-            init($element, settings);
+            $element.select2({
+                ajax: {
+                    data: function(params) {
+                        return {
+                            term: params.term,
+                            page: params.page,
+                            app_label: $element.data('app-label'),
+                            model_name: $element.data('model-name'),
+                            field_name: $element.data('field-name')
+                        };
+                    }
```

Comment:
As the admin uses es6, [this could use the newer arrow syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions). NBD either way.

```suggestion
                    data: (params) => {
                        return {
                            term: params.term,
                            page: params.page,
                            app_label: $element.data('app-label'),
                            model_name: $element.data('model-name'),
                            field_name: $element.data('field-name')
                        };
                    }
```